tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -132,10 +152,6 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
         c2r.sendline("n")
     assert c2r.exitstatus != 0
 
-    # Clean up
-    os.system(f"cp /tmp/__init__.py {PATH_TO_VERSION}")
-    del os.environ["CONVERT2RHEL_ALLOW_OLDER_VERSION"]
```

Comment:
I would personally keep the `del os.environ["CONVERT2RHEL_ALLOW_OLDER_VERSION"]` here instead of in the `_restore_c2r_version()` as it is not something I would expect the "restore version" to do.

Reply:
That is a valid point, however I would rather keep it in the fixture.
So I will take it out of the `_restore_c2r_version()` function and put it after the function call, WDYT @kokesak ?

Reply:
On a second thought and after a discussion, we will keep the environment variable actions (setting and deleting) inside the test for the sake of readability.
It doesn't really make much sense to have the envar set in the test (or even in the test metadata, which was contemplated for the future) and deleted by the fixture.

Reply:
Final thought.
Create a simple fixture to handle the envar.
In case the test fails, the removal will be handled anyway, not interfering with subsequent tests.

Reply:
Alright, why not @danmyway